tei_adapter/tei_updater/src/tei_updater.py

```diff
@@ -57,19 +59,28 @@ def diff_trees(old_tree, new_tree):
             get_path_from_diff(changed.up.path()) for changed in values_changed
         }
         messages += [
-            {"path": path, "url": new_tree[path]["url"]} for path in paths_changed
+            {
+                "path": path,
+                "uri": new_tree[path]["uri"],
+                "timeModified": time.isoformat(),
+            }
+            for path in paths_changed
         ]
     if items_added:
         messages += [
             {
                 "path": get_path_from_diff(added.path()),
-                "url": new_tree[get_path_from_diff(added.path())]["url"],
+                "uri": new_tree[get_path_from_diff(added.path())]["uri"],
+                "timeModified": time.isoformat(),
```

Comment:
Bikeshed - would it be better to have `"status": "deleted | modified | etc"` that you can differentiate on, and just have a "time" attribute?

Reply:
I don't really mind to be honest. I have a slight preference for different messages rather than statuses with different values because in the scala side you can leave it up to circe to decode into one or the other and then pattern match on the type. But you can pattern match on the value of status as well so 🤷‍♀️